cybered/hashing/templates/hashing/tools.html

```diff
@@ -0,0 +1,86 @@
+<!doctype HTML>
+<html>
+{% include "includes/head.html" %}
+
+<body>
+    {% include "includes/header.html" %}
+    <div class="container">
+        <h3>Tools</h3>
+
+        <p>
+            Enter a message and press the 'Hash' button to see the message digests for the message.
+            If you enter a password, then you will also see the keyed digests.
+        </p>
+
+        {% if input_text %}
+        <div class="card">
+            <div class="card-body">
+                <h5 class="card-title">Original Text</h5>
+                <div style="max-height: 100px; overflow-y: auto;">
+                    {{ input_text }}
+                </div>
+            </div>
+            {% if input_key %}
+            <div class="card-body">
+                <h5 class="card-title">Password</h5>
+                <div style="max-height: 100px; overflow-y: auto;">
+                    {{ input_key }}
+                </div>
+            </div>
+            <div class="card-body">
+                <h5 class="card-title">Combined Text and Password</h5>
+                <div style="max-height: 100px; overflow-y: auto;">
+                    {{ keyed_text }}
+                </div>
+            </div>
+            {% endif %}
+        </div>
+
+        <div class="table-responsive">
```

Comment:
I really like how this turned out!

Reply:
Me too! If you view it on a small screen it'll become horizontal scrollable; not sure if you saw that...

Reply:
It's too large for the container on a large screen too..